exasol_advanced_analytics_framework/lua/src/query_loop.lua

```diff
@@ -69,6 +69,25 @@ function M.prepare_init_query(arguments, meta)
     return query
 end
 
+local FIRST_COLUMN_INDEX <const> = 1
+
+function _handle_query_error(query, result, exa_env)
```

Comment:
If you use module style, you could make the internal functions local (unless you plan white box tests, that is).